tests/integration-tests/tests/pyxis/test_pyxis/test_pyxis/pcluster.config.yaml

```diff
@@ -0,0 +1,27 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+  Scheduler: {{ scheduler }}
+  SlurmQueues:
+    - Name: queue-0
+      ComputeResources:
+        - Name: compute-resource-0
+          Instances:
+            - InstanceType: t3.small
+          MinCount: 0
+          MaxCount: 1000
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+SharedStorage:
+  - MountDir: {{ mount_dir }}
```

Comment:
The EFs storage should be used as internal shared storage to replicate the race condition.

Reply:
Fixed